### DISCUSSION TITLE
Timeline for SDK's

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1620] [General]
Is there a timeline yet for SDK's for developers?
Specifically those for Themes and Blocks.

And when is the next webinar?

**Comment by meetmandeep** [comment]
Next webinar is on Aug 18th, 2021; you may register now at https://www.vanjaro.com/resources/learn/webinars

We'll have our first webinar on setting up the dev environment in September with following webinars focused on extending Vanjaro.
